### PR TITLE
Tool modifications

### PR DESCRIPTION
**Pull request by mbeacom**
##### Description
The goal of this PR is to optimize the new tool voting queries and specify categories on `dashboard.models.Tool`

This PR reduces the DB queries for the tools page from 43 to 25 and cuts the response time just about in half.

##### Checklist
- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
tools

##### Testing
Tested locally.

### CODE REVIEW

File: app/dashboard/views.py [hunk 1]
```diff
@@ -824,56 +824,58 @@ def apitos(request):
...
+        "description": _("Gitcoin is built using Gitcoin.  Purdy cool, huh? "),
+        "tools": tools.exclude(category__ne=Tool.CAT_BUILD)
+    }, {
+        "title": _("Tools in Alpha"),
+        "description": _("These fresh new tools are looking for someone to test ride them!"),
+        "tools": tools.exclude(category__ne=Tool.CAT_ALPHA)
+    }, {
+        "title": _("Tools Coming Soon"),
+        "description": _("These tools will be ready soon.  They'll get here sooner if you help BUIDL them :)"),
+        "tools": tools.exclude(category__ne=Tool.CAT_COMING_SOON)
```
Comment (owocki) [inline app/dashboard/views.py]: wouldnt this be `tools.filter` not `tools.exclude`?

File: app/dashboard/views.py [hunk 2]
```diff
@@ -824,56 +824,58 @@ def apitos(request):
...
         actors[key]['slug'] = slugify(actors[key]['title'])
 
     profile_id = request.user.profile.pk if request.user.is_authenticated and hasattr(request.user, 'profile') else None
-    ups = list(ToolVote.objects.filter(profile_id = profile_id, value = 1).values_list('tool', flat=True))
-    profile_up_votes_tool_ids = ','.join(str(x) for x in ups)
-    downs = list(ToolVote.objects.filter(profile_id = profile_id, value = -1).values_list('tool', flat=True))
-    profile_down_votes_tool_ids = ','.join(str(x) for x in downs)
+    if profile_id:
+        ups = list(request.user.profile.votes.filter(value=1).values_list('tool', flat=True))
+        # ups = list(ToolVote.objects.filter(profile_id=profile_id, value=1).values_list('tool', flat=True))
```
Comment (owocki) [inline app/dashboard/views.py]: dead code comment

File: app/dashboard/views.py (hunk 1 above)
Comment (owocki) [reply app/dashboard/views.py]: oh i guess the _ne takes care of that... but feels like a double negative to me

Comment (mbeacom) [reply app/dashboard/views.py]: @owocki `QuerySet.exclude` with `ne` reduces the query results from `.all()` - I'll switch it back to filter, though.

**Pull request merged**